### ISSUE TITLE
Peer dropouts

### ISSUE THREAD

**Issue by deceptivestudios** [paritytech/parity#7861]
> I'm running:
>
> - **Which Parity version?**: 1.9.2
> - **Which operating system?**: Linux CentOS 7 VM
> - **How installed?**: RPM
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

**Actual**
After a warp sync I was within about 100,000 of the current block so restarted with my config file as a systemctl service. I had consistent peer connections 128+ up until synchronization and then the peer count dropped to less than 10 (fluctuates wildly from 1 to 20). During the sync, I was consistently receiving blocks with an average of one per 3 seconds, but now that I am in sync the last block I received was more than 5 minutes ago.

Restarting the node kicks off the syncronization again, and until I reach the latest block everything is consistent. When the latest block is imported the peer count drops and the node goes back into this loop of falling behind.

**Expected Behaviour**
After synchronization, peer connections stay high and blocks are received consistently.

**Steps to Reproduce**
I have performed a full block sync multiple times and each time I have finalized a sync this happens. I had noticed that this was apparently fixed with previous builds and so I have updated to the latest version.

Parity initially run with this command:
`parity --mode passive --db-compaction ssd --pruning fast --warp --cache-size 4096 --db-path /home/parity/io.parity.ethereum/`

After warp achieved, the following configuration file is used (db path is moved to correct location before starting):
```
[parity]
auto_update = "all"
base_path = "/home/parity/io.parity.ethereum"
identity = "Ethereum Node"
mode = "active"

[account]
unlock = ["###"]
password = ["/etc/parity/.passwd"]

[ui]
interface = "0.0.0.0"
path = "/home/parity/io.parity.ethereum/signer"

[rpc]
interface = "0.0.0.0"
apis = ["web3", "eth", "net", "parity", "traces", "rpc", "secretstore", "parity_accounts", "personal", "signer"]

[websockets]
interface = "0.0.0.0"

[mining]
author = "###"

[stratum]
interface = "0.0.0.0"

[network]
min_peers = 128
max_peers = 256
nat = "extip:xxx.xxx.xxx.xxx"
discovery = true
allow_ips = "all"
max_pending_peers = 64
```

**Comment by 5chdn** [comment]
Why are you running the initial synchronization with `--mode passive` ?

Probably duplicate of #7531
